lib/index.js

```diff
@@ -169,6 +168,7 @@ export default {
         const execArgs = ['-lc', cmd];
         const execOpts = {
           cwd: dirname(configurationFilePath),
+          timeout: 0,
```

Comment:
You will need to observe the setting up in `activate()`, then conditionally add it [like this](https://github.com/AtomLinter/linter-flake8/blob/ab20e7c25a74f6d76607de2229834b97880e1aef/lib/main.js#L181-L183).